src/main/java/com/github/games647/craftapi/resolver/MojangResolver.java

```
@@ -191,6 +193,16 @@ protected Optional<Profile> findProfile(HttpClient client, HttpRequest req)
                 return findProfile(proxyClient, req);
             }
 
+            if (responseCode == HttpURLConnection.HTTP_FORBIDDEN) {
+                if (useBackupUuidUrl) {
+                    return Optional.empty();
```

Wasnt really sure what to do if both urls respond with 403, as if id throw a new error i would need to update it much further up the chain.

---

I added an `IOException` now. I think it's better to not ignore the error silently.